pyapp/__main__.py

```diff
@@ -0,0 +1,20 @@
+import time
+import os
+from .data_collection import create_guage,start_server,process_request,data_publish
+
+
+if __name__ == '__main__':
+    # run till infinity 
+    print('created metric')
```

Comment:
Consider replacing `print` statements with Python logging. Enabled us to have various log levels along with formatting https://docs.python.org/3/howto/logging.html